### PR TITLE
Add cloudflare peer IDs

### PR DESCRIPTION
**Pull request by mathew-cf**

### CODE REVIEW

**Review by BigLep — APPROVED** [review]

File: docs/how-to/peering-with-content-providers.md [hunk 1]
```diff
@@ -44,7 +44,19 @@ To have your platform added to this list, please [open a PR to edit this page](h
 
 |Peer ID|Addresses|
 |:-|:-|
-|`QmcfgsJsMtx6qJb74akCw1M24X1zFwgGo11h1cuhwQjtJP`|`/ip6/2606:4700:60::6/tcp/4009` <br/><br/>`/ip4/172.65.0.13/tcp/4009`|
+|`QmcFf2FH3CEgTNHeMRGhN7HNHU1EXAxoEk6EFuSyXCsvRE`|`/dnsaddr/bootstrap.cloudflare-ipfs.com/ipfs/QmcFf2FH3CEgTNHeMRGhN7HNHU1EXAxoEk6EFuSyXCsvRE`|
```
[the remark this answers is not in the document]
Comment (mathew-cf) [reply docs/how-to/peering-with-content-providers.md:47]: I'm still working on opening up ingress to the IP. May have been a bit too early with this PR

**Comment by johnnymatthews** [comment]
Looks like you're still working on this. I'm gonna convert it to a draft. Ping folks when you're ready for a review.

**Comment by mathew-cf** [comment]
This is ready for review. Our peer count limit is fairly low as of right now but you should be able to ping our nodes without issue
```
$ ipfs ping /dnsaddr/node-12.ingress.cloudflare-ipfs.com/ipfs/QmcfV2sg9zaq7UUHVCGuSvT2M2rnLBAPsiE79vVyK3Cuev
PING QmcfV2sg9zaq7UUHVCGuSvT2M2rnLBAPsiE79vVyK3Cuev.
Pong received: time=70.94 ms
Pong received: time=94.22 ms
Pong received: time=83.20 ms
Pong received: time=74.81 ms
Pong received: time=74.89 ms
Pong received: time=79.82 ms
Pong received: time=75.27 ms
Pong received: time=71.42 ms
Pong received: time=70.23 ms
Pong received: time=80.62 ms
Average latency: 77.54ms
```

**Comment by johnnymatthews** [comment]
Nice. I tried pinging (`ipfs ping ...`) each of them, and got a pong from most.

File: docs/how-to/peering-with-content-providers.md [hunk 2]
```diff
@@ -42,9 +42,20 @@ To have your platform added to this list, please [open a PR to edit this page](h
 
 ### Cloudflare
 
-| Peer ID | Addresses |
-| ------- | --------- |
-|`QmcfgsJsMtx6qJb74akCw1M24X1zFwgGo11h1cuhwQjtJP`|`/ip6/2606:4700:60::6/tcp/4009` <br/><br/>`/ip4/172.65.0.13/tcp/4009`|
+| Peer ID                                          | Addresses                                      |
+| -----------------------------------------------  | ---------------------------------------------  |
+| `QmcFf2FH3CEgTNHeMRGhN7HNHU1EXAxoEk6EFuSyXCsvRE` | `/dnsaddr/node-1.ingress.cloudflare-ipfs.com`  |
+| `QmcFmLd5ySfk2WZuJ1mfSWLDjdmHZq7rSAua4GoeSQfs1z` | `/dnsaddr/node-2.ingress.cloudflare-ipfs.com`  |
```
Comment (johnnymatthews) [inline docs/how-to/peering-with-content-providers.md:48]: Didn't get a response for this one.

File: docs/how-to/peering-with-content-providers.md [hunk 3]
```diff
@@ -42,9 +42,20 @@ To have your platform added to this list, please [open a PR to edit this page](h
...
+| `QmcfFmzSDVbwexQ9Au2pt5YEXHK5xajwgaU6PpkbLWerMa` | `/dnsaddr/node-3.ingress.cloudflare-ipfs.com`  |
+| `QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx` | `/dnsaddr/node-4.ingress.cloudflare-ipfs.com`  |
```
Comment (johnnymatthews) [inline docs/how-to/peering-with-content-providers.md:50]: Didn't get a response for this one.

File: docs/how-to/peering-with-content-providers.md [hunk 4]
```diff
@@ -42,9 +42,20 @@ To have your platform added to this list, please [open a PR to edit this page](h
...
+| `QmcfVvzK4tMdFmpJjEKDUoqRgP4W9FnmJoziYX5GXJJ8eZ` | `/dnsaddr/node-5.ingress.cloudflare-ipfs.com`  |
+| `QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G` | `/dnsaddr/node-6.ingress.cloudflare-ipfs.com`  |
```
Comment (johnnymatthews) [inline docs/how-to/peering-with-content-providers.md:52]: Didn't get a response for this one.

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
I can't reach these three nodes:

```
QmcFmLd5ySfk2WZuJ1mfSWLDjdmHZq7rSAua4GoeSQfs1z
QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx
QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G
```

Are they actually up? I'm at an airport, so there's a chance that the WiFi here is bugging out.

**Comment by isaadabbasi** [comment]
> I can't reach these three nodes:
> 
> ```
> QmcFmLd5ySfk2WZuJ1mfSWLDjdmHZq7rSAua4GoeSQfs1z
> QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx
> QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G
> ```
> 

I verified and still these 3 nodes are unreachable. Any update on this?

**Comment by johnnymatthews** [comment]
> > I can't reach these three nodes:
> > ```
> > QmcFmLd5ySfk2WZuJ1mfSWLDjdmHZq7rSAua4GoeSQfs1z
> > QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx
> > QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G
> > ```
> 
> I verified and still these 3 nodes are unreachable. Any update on this?

I just tried this again. The first and last gave me a pong! But the middle (`QmcfJ...Uswx`) didnt:

```shell
$ › ipfs ping QmcFmLd5ySfk2WZuJ1mfSWLDjdmHZq7rSAua4GoeSQfs1z
PING QmcFmLd5ySfk2WZuJ1mfSWLDjdmHZq7rSAua4GoeSQfs1z.
Pong received: time=145.87 ms
$ › ipfs ping QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx
Looking up peer QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx
PING QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx.
Ping error: failed to dial QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx:
  * [/ip4/136.144.57.127/tcp/4002/ws/p2p/12D3KooWJbARcvvEEF4AAqvAEaVYRkEUNPC3Rv3joebqfPh4LaKq/p2p-circuit] dial backoff
  * [/ip6/2604:1380:45f1:4d00::3/tcp/4002/ws/p2p/12D3KooWJbARcvvEEF4AAqvAEaVYRkEUNPC3Rv3joebqfPh4LaKq/p2p-circuit] error opening hop stream to relay: context deadline exceeded
  * [/ip4/136.144.57.127/tcp/4001/p2p/12D3KooWJbARcvvEEF4AAqvAEaVYRkEUNPC3Rv3joebqfPh4LaKq/p2p-circuit] error opening relay circuit: NO_RESERVATION (204)
  * [/ip6/2604:1380:45f1:4d00::3/tcp/4001/p2p/12D3KooWJbARcvvEEF4AAqvAEaVYRkEUNPC3Rv3joebqfPh4LaKq/p2p-circuit] concurrent active dial through the same relay failed with a protocol error
Error: ping failed
$ › ipfs ping QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G
Looking up peer QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G
PING QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G.
Pong received: time=85.61 ms
```

**Comment by 2color** [comment]
Just ran a test to ping all the nodes and they were all reachable 

I got a few `Error: peer lookup failed: routing: not found` but after trying a couple of times it worked and I got a `Pong received`.

| Peer ID                                          | ipfs ping peerId |
| ------------------------------------------------ | ---------------- |
| `QmcFf2FH3CEgTNHeMRGhN7HNHU1EXAxoEk6EFuSyXCsvRE` | ✅               |
| `QmcFmLd5ySfk2WZuJ1mfSWLDjdmHZq7rSAua4GoeSQfs1z` | ✅               |
| `QmcfFmzSDVbwexQ9Au2pt5YEXHK5xajwgaU6PpkbLWerMa` | ✅               |
| `QmcfJeB3Js1FG7T8YaZATEiaHqNKVdQfybYYkbT1knUswx` | ✅               |
| `QmcfVvzK4tMdFmpJjEKDUoqRgP4W9FnmJoziYX5GXJJ8eZ` | ✅               |
| `QmcfZD3VKrUxyP9BbyUnZDpbqDnT7cQ4WjPP8TRLXaoE7G` | ✅               |
| `QmcfZP2LuW4jxviTeG8fi28qjnZScACb8PEgHAc17ZEri3` | ✅               |
| `QmcfgsJsMtx6qJb74akCw1M24X1zFwgGo11h1cuhwQjtJP` | ✅               |
| `Qmcfr2FC7pFzJbTSDfYaSy1J8Uuy8ccGLeLyqJCKJvTHMi` | ✅               |
| `QmcfR3V5YAtHBzxVACWCzXTt26SyEkxdwhGJ6875A8BuWx` | ✅               |
| `Qmcfuo1TM9uUiJp6dTbm915Rf1aTqm3a3dnmCdDQLHgvL5` | ✅               |
| `QmcfV2sg9zaq7UUHVCGuSvT2M2rnLBAPsiE79vVyK3Cuev` | ✅               |

**Review by johnnymatthews — APPROVED** [review]

**Comment by johnnymatthews** [comment]
Tried all the peers again and got responses from everything! Merging!

**Pull request merged**